Snowball-Finance/axial-frontend
# liquidity page design

To test:
1. Check the design for navigation with respect to figma
2. Check the design for list of liquidity pools
3. Check the **balance and total** of a pool in listing and deposit, withdraw **my share** section
4. Check the deposit and withdraw page design
5. Check the deposit and with review modal with design and ui-integration.
